assignments/array-methods.js

```diff
@@ -55,29 +55,47 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
-console.log(fullName);
+let fullName = [first_name,last_name.foreach(currentValue => {
+
+console.log(fullName);})];
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
-console.log(allCaps); 
+let allCaps = [(item,first_name) =>{
+    $;{item.first_name.UpperCase()}; 
+}];
+console.log(UpperCase); 
 
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
-let largeShirts = [];
+let largeShirts = [shirt_size.filter(item =>{
```

Comment:
```
console output: shirt-size is not defined
```

Array methods are methods on the array. which means you need to use a dot notation on the array that is given which is `runners.filter` `intead of shirt_size.filter` since you are filtering through the whole entire array.